src/redux/books/books.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console */
```

Comment:
Good job working on the project requirements but it is against microverse policy to disable linters because linters help us write clean code. It would be nice to enable eslint.